MediaBrowser.Controller/MediaEncoding/EncodingHelper.cs

```diff
@@ -1582,11 +1605,25 @@ public string GetGraphicalSubtitleParam(EncodingJobInfo state, EncodingOptions o
                 ? 0
                 : state.SubtitleStream.Index;
 
+	    var videoDecoder = GetHardwareAcceleratedVideoDecoder(state, options);
+
+	    var retStr = " -filter_complex \"[{0}:{1}]{4}[sub];[0:{2}][sub]overlay{3}\"";
```

Comment:
please make a comment about what this magic does (I assume it's for burning in the subtitles, but still)

Reply:
I've made more commits to my jellyfin fork/master branch but I'm not sure if its possible to change without a new pull request (and cancelling this one)